entrypoint.sh

```diff
@@ -74,8 +84,9 @@ _push_to_github() {
         else
              git push origin
         fi
-
     else
+        echo "::debug::Push commit to remote branch $INPUT_BRANCH"
+        git pull origin "HEAD:$INPUT_BRANCH"
```

Comment:
This should be git-push, not git-pull. It's the merge triggered by the pull that is causing the error.

Also, it's probably safer to do `git push origin "HEAD:/refs/heads/$INPUT_BRANCH"`. (We know that that input branch is a head ref, not a tag ref so we should be explicit to avoid any chance of name collision.)